README.md

```diff
@@ -2,11 +2,11 @@
     <a href="http://poliastro.github.io/"><image src="http://poliastro.github.io/images/logo_text.png" alt="poliastro logo" width="675px"></a>
 </p>
 
-Name:         | poliastro
-:------------:|:--------------
-**Website**:  | <https://poliastro.github.io/>
-**Author**:   | Juan Luis Cano Rodríguez [![orcid](https://img.shields.io/badge/id-0000--0002--2187--161X-a6ce39.svg)](http://orcid.org/0000-0002-2187-161X)
-**Version**:  | 0.15.dev0
+|    Name:     | poliastro                                                                                                                                    |
```

Comment:
Hello @AlphaVictorDelta ... I don't think this needs to be changed as it is already configured correctly ... Can you please revert this back?

Reply:
No changes can be seen 🤔